ostrica/utilities/header.cfg

```diff
@@ -103,6 +103,6 @@ ul {
                 'opacity': 0.25,
                     'text-opacity': 0
             }),
-            maxZoom: 10,
-            minZoom: 0.25,
+            //maxZoom: 10,
```

Comment:
There is no need to add these details at the moment. Maybe we can give this option to the user by adding input boxes on the Web UI.